### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by nbiton**
It doesn't look like I successfully squashed the commits... not sure why it didn't go well.
This PR includes working signup and login pages

### CODE REVIEW

**Comment by kaihendry** [comment]
Doesn't seem to build minified btw:
```
~/lmb/unee-t-fe$ meteor build ../output
Errors prevented bundling:
While minifying app stylesheet:
packages/minifier-css.js:29:29: traverse[node.type] is not a function
at visit (packages/minifier-css.js:29:29)
at packages/minifier-css.js:103:43
at Array.map (native)
at stringifyAst (packages/minifier-css.js:103:14)
at Array.map (native)
at MinifyAst (packages/minifier-css.js:107:18)
at Object.CssTools.minifyCssAst (packages/minifier-css.js:256:12)
at Object.CssTools.minifyCss (packages/minifier-css.js:253:21)
at CssToolsMinifier.processFilesForBundle (packages/minifier-postcss/plugin/minify-css.js:120:34)


~/lmb/unee-t-fe$ git describe --always
114e669
```

**Comment by kaihendry** [comment]
Hoping @sebdeckers will review..

File: imports/api/claims.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   - changed(collection-name, id, attributes) - changes an existing document and modifies the attributes
+   - removed(collection-name, id) - removed an existing document by id
+   - ready() - notifies the subscribed client of the initial success of the subscription with the initial data
+   - onStop(callback) - adds a handler for when a subscribed client removes its subscription
+   */
+  Meteor.publish('claims', function () {
+    if (!this.userId) return
+    const currUser = Meteor.users.findOne({_id: this.userId})
+    // Fetching simple data
+    callAPI('get', '/rest/bug/21', {token: currUser.bugzillaCreds.token})
```
Comment (sebdeckers) [inline imports/api/claims.js]: @nbiton Why `21`?

File: imports/ui/todo/app.mss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    background: transparent;
+    border: none;
+    width: 100%;
+    padding-right: 80px;
+    font-size: 1em;
+    &:focus {
+      outline: 0;
+    }
+  }
+} .list {
```
Comment (sebdeckers) [inline imports/ui/todo/app.mss]: Weird whitespace/line, is this a typo?

File: imports/api/claims.js (hunk 1 above)
Comment (nbiton) [reply imports/api/claims.js]: This is just a dummy API request. It just so happened that id 21 was an existing claim on my own bugzilla instance...

File: imports/ui/todo/app.mss (hunk 2 above)
Comment (nbiton) [reply imports/ui/todo/app.mss]: Yup, looks like a typo

File: imports/api/claims.js (hunk 1 above)
Comment (sebdeckers) [reply imports/api/claims.js]: Ahh, thanks for explaining. How about using a placeholder variable that explicitly labels it as a dummy?

**Pull request merged**